README.md

```diff
@@ -1,87 +1,71 @@
 
-### Project context
+Interested in joining our team? See our [contribution guide](CONTRIBUTING.md) to get started!
 
-This project is part of a larger initative at Hack for LA around creating a shared housing application, that can be used by organizations accross the county to help people tranisition to living independently and sustainably through host homes, empty bedrooms, and roomate matching for shared housing with lease signing.
+## Project Context
 
-## Technology used
+This project is part of a larger initiative at Hack for LA around creating a shared housing application, that can be used by organizations across the county to help people transition to living independently and sustainably through host homes, empty bedrooms, and roommate matching for shared housing with lease signing.
 
-### Frontend
-- [TypeScript](https://www.typescriptlang.org/)
-- [React](https://reactjs.org/docs/getting-started.html)
-- [Redux Toolkit](https://redux-toolkit.js.org/)
-- [Material UI](https://material-ui.com/)
-- [Vite](https://vitejs.dev/)
+## Technology Overview
 
-### Backend
-- [Python/Flask](https://flask.palletsprojects.com/en/1.1.x/)
-- [PostgreSQL](https://www.postgresql.org/)
-- [SQLAlchemy](https://www.sqlalchemy.org/)
-- [Docker](https://docs.docker.com/)
-- [Nginx](https://nginx.org/en/docs/)
-- [AWS](https://docs.aws.amazon.com/)
+The HomeUniteUs project is structured as a multi-[docker](https://docs.docker.com/) container application, with secret-protected access to networked resources. The project contains three containers, whose activities are coordinated using the `docker compose` configuration outlined in `docker-compose.yml`. The three containers are:
 
-# How to contribute
+1. `app`: A frontend [React](https://reactjs.org/docs/getting-started.html) app developed using [TypeScript](https://www.typescriptlang.org/).
+   * We use [Redux](https://redux.js.org/) to manage client state, with the [Redux Toolkit](https://redux-toolkit.js.org/) to simplify development.
+   * We use the [Material UI](https://material-ui.com/) component library, for access to high quality UI components designed with accessibility in mind.
+   * We use the [Vite](https://vitejs.dev/) build tool, for fast dev builds and optimized production builds.
+2. `api`: A backend python [Flask](https://flask.palletsprojects.com/en/1.1.x/) REST API, hosted on [AWS](https://docs.aws.amazon.com/).
+   * We use [connexion](https://connexion.readthedocs.io/en/latest/) to simplify our API specification and validation. `connexion` uses a .yaml API specification to map API URLs to specific python functions within our `Flask` app, and will handle routing and parameter passing when an API request is made. `connexion` also generates documentation for our API, and provides a useful user interface (at `{URL}/api/ui`) that can be to easily interact with the API for development purposes.
```

Comment:
`connexion` uses the OpenAPI specification to map API URLs to specific python functions within our `Flask` app, and will handle routing, validation , security, and parameter passing when an API request is made.

Reply:
Thanks! Added links and more complete description in [ebfba50](https://github.com/hackforla/HomeUniteUs/pull/557/commits/ebfba503f84ed549893cd2f68623dc90597fdafe)